vhdbuilder/packer/install-dependencies.sh

```diff
@@ -55,6 +55,8 @@ EOF
 fi
 capture_benchmark "purge_and_reinstall_ubuntu"
 
+installOras
```

Comment:
It will be great if we can move the logic to the package for loop (around ln186). You can find other packages installed there.